sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -366,6 +368,51 @@ def test_model_pipeline_same_as_normalize_true(LinearModel, params):
     assert_allclose(y_pred_normalize, y_pred_standardize)
 
 
+# FIXME: 'normalize' to be removed in 1.2
+@pytest.mark.filterwarnings("ignore:'normalize' was deprecated")
+@pytest.mark.parametrize(
+    "estimator, params, is_sparse",
+    [(LinearRegression, {}, True),
+     (LinearRegression, {}, False)
+     ]
+)
+def test_linear_model_sample_weights_normalize_in_pipeline(estimator,
+                                                           params,
+                                                           is_sparse):
+    rng = np.random.RandomState(0)
+
+    # It would not work with under-determined systems
+    for n_samples, n_features in ((6, 5), ):
+
+        y = rng.randn(n_samples)
+        X = rng.randn(n_samples, n_features)
+        X_test = rng.randn(n_samples, n_features)
+        if is_sparse:
+            X = sparse.csr_matrix(X)
+            X_test = sparse.csr_matrix(X)
+
+        sample_weight = 1.0 + rng.rand(n_samples)
```

Comment:
it is not clear at a first glance why we had 1.0 to the `sample_weight`. Is there any reason? If yes, we should probably have a comment on the top.

Reply:
I think it should work without the +1, no? If so, let's just remove it. If it's needed, we indeed need a comment to explain why.